app_server/.gitignore

```diff
@@ -29,3 +29,4 @@ build/Release
 node_modules
 doc
 report
+.tern-project
```

Comment:
Use your local `exclude` file instead if it's not relevant to the all contributors.